pkg/apis/pipeline/v1/pipeline_validation.go

```diff
@@ -184,13 +184,25 @@ func validatePipelineContextVariables(tasks []PipelineTask) *apis.FieldError {
 	var paramValues []string
 	for _, task := range tasks {
 		var matrixParams []Param
+		var includeParams []Param
```

Comment:
Just having a discussion in other PR #6180, we could introduce a new type and use that instead of `[]Param`:

```
type Params []Param
```

Reply:
Not a blocker for this PR though!

Reply:
Hey @EmmaMunley, please make sure to follow up on this once PR #6180 is resolved.